### PR TITLE
Avoid updating pull requests status for verification in non-blocking runs

### PR DESCRIPTION
**Pull request by AlexMSmithCA**
Related to [probot-app-workflow PR#129](https://github.com/fusionjs/probot-app-workflow/pull/129) that runs Fusion.js verification on all PRs.

### CODE REVIEW

File: src/afterVerification.js [hunk 1]
```diff
@@ -26,12 +26,12 @@ octokit.authenticate({
 
 module.exports = async function afterVerification() {
   const statusMetadata = shelljs.exec('buildkite-agent meta-data get "status"');
-  const isPrerelease = shelljs.exec(
-    'buildkite-agent meta-data get "prerelease"'
```
Comment (AlexMSmithCA) [inline src/afterVerification.js]: Where is this metadata field generated?

File: src/afterVerification.js [hunk 2]
```diff
@@ -26,12 +26,12 @@ octokit.authenticate({
...
+  const isNonBlocking = shelljs.exec(
+    'buildkite-agent meta-data get "release-pr-non-blocking"'
```
Comment (KevinGrandon) [inline src/afterVerification.js]: `release-pr-non-blocking` naming feels a little strange to me. Does this mean this is a release PR or not? Perhaps we should only set a single status, `is-release-pr` which signals that the status is required, and is only used for releases?

Comment (AlexMSmithCA) [reply src/afterVerification.js]: :+1: I like that idea.  I was using the format that was already there, but I agree that makes it a little confusing.

Ideally we would also rename this to something like "fusionjs-verification" as opposed to "release-verification"

**Comment by AlexMSmithCA** [comment]
!merge

**Pull request merged**